package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "writers-haven-api",
+  "version": "1.0.0",
+  "description": "",
```

Comment:
@Duncanian please add a description here.

Reply:
@samuelkubai , I have added it.